src/AudioUnit/AudioComponent.cs

```diff
@@ -342,6 +351,13 @@ public UIKit.UIImage GetIcon (float desiredPointSize)
 				return AudioComponentGetLastActiveTime (handle);
 			}
 		}
+
+		[Watch (7,0), TV (14,0), iOS (14,0)]
+		public UIImage CopyIcon () {
```

Comment:
minor
* style, `{` on the next line
* maybe `GetIcon` since we don't have to follow the ObjC copy rule/convention

Reply:
We already have a GetIcon, either we ignore the method of we use Copy